scripts/runtest.sh

```diff
@@ -9,35 +9,52 @@ then
   exit 1
 fi
 
+DOMAIN="http://loadtest.dev"
+
+LOCUST=/usr/local/bin/locust
+LOCUST_PLAN=/root/drupal-loadtest/locust_testplan.py
+
+RESTART_MYSQL="/usr/local/etc/rc.d/mysql-server restart"
```

Comment:
I don't have an EL7 server to ssh into for these paths, if you can provide them I'm happy to add them.